views/dashboard/index.html

```diff
@@ -22,7 +22,7 @@ <h2 style="width: 800px; display: inline;">{{=selected_chapter['chapter_name']}}
 					  {{pass}}
 					  {{pass}}
 					</select>
-					<button onclick="window.location.href='grades'">Gradebook</button>
+					<button onclick="window.location.href+='/grades'">Gradebook</button>
```

Comment:
@bjones I think this is a typo `+=` breaks the gradebook link.  After multiple clicks I end up with a URL like:  ``http://127.0.0.1:8000/runestone/dashboard/index/grades/grades/grades/grades/grades``

Reply:
This change just needs to be reverted back to what it was.  Was it actually broken for you??  The old way was fine on runestone.academy and on my test machine.

Reply:
I'm reverting it in the test branch and will accept the pr with that change.

Reply:
It now works for me -- earlier, the link did't work at all. ??? Anyway, thanks!